docs/source/usage.rst

```diff
@@ -16,6 +19,48 @@ In the template (e.g. post_detail.) add the following template tags where ``obj`
     {% render_comments obj request %}  {# Render all the comments belong to the passed object "obj" #}
 
 
+2. Displaying Comment Count in Post Page:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In ``views.py`` under the post view add the following line at the top of your page:
+
+.. code:: python
+
+    from comment.models import Comment
+    
+    ...
+    def post_detail(request, pk):
+    post = get_object_or_404(Post, pk=pk)
+    comments = Comment.objects.filter(content_type_id=1, object_id=post.id)
+    
+    context = {'post':post, 'comments':comments}
+    ...
+
+
+Here, ``content_type_id`` refers to the model that your comments are related to. (e.g. you may have different ids if you have seperate models for posts or categorys of product etc)
+Check the ``object id`` under the comments app in the admin page for the correct id to a comment.
+
+
+In the template (e.g. post_detail.html), add the following tag
+
+.. code:: jinja
+
+    {{ comments.count }}
+
+
```

Comment:
I think you can remove this. Comment count can be displayed inside any template by just using:

```jinja
{% get_comments_count obj user %}
```
There is no need to pass something extra from the `view` function.

Reply:
This is actually what _you_ asked me to write documentation for:
https://github.com/Radi85/Comment/issues/82#issuecomment-698262773

Reply:
The content under https://github.com/Radi85/Comment/pull/84/files#diff-7d3008165b82862fcdec94df0ff555feR51 is sufficient. Just the use of the template tag `get_comments_count` needed to be mentioned.

Reply:
how do i can change the default ordering comments for the products or content types in my project????
i want the comments be ordered by oldest to the newest